src/components/lists/ArticleList.jsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react';
+import Article from './list-items/Article';
+import { getArticles } from '../../utils';
+
+export default function ArticleList() {
+	const [articleList, setArticleList] = useState([]);
+	useEffect(() => {
+		getArticles().then((response) => {
+			setArticleList(response);
+		});
+	}, []);
+
+	return (
+		<ol>
+			{articleList.map((article, index) => {
+				return (
+					<li key={article + index}>
+						<Article article={article} />
```

Comment:
consider renaming this 'article card' - it'll make it easier later on to differentiate between single articles and article cards :)